migrations/2_deploy_contracts.js

```diff
@@ -1,12 +1,9 @@
 var RelayHub = artifacts.require('./RelayHub.sol')
-var RLPReader = artifacts.require('./RLPReader.sol')
 var StakeManager = artifacts.require('./StakeManager.sol')
 var Penalizer = artifacts.require('./Penalizer.sol')
 
-module.exports = function (deployer) {
-  deployer.deploy(RLPReader)
-  deployer.link(RLPReader, RelayHub)
-  deployer.deploy(RelayHub, 16, '0x0000000000000000000000000000000000000000', '0x0000000000000000000000000000000000000000')
-  deployer.deploy(StakeManager)
-  deployer.deploy(Penalizer)
+module.exports = async function (deployer) {
+  await deployer.deploy(StakeManager)
+  await deployer.deploy(Penalizer)
+  await deployer.deploy(RelayHub, 16, StakeManager.address, Penalizer.address)
```

Comment:
We never used linking: our libs were always "internal" functions.

Thinking about it, we __could__ implement Penalizer, StakeManager as libraries, instead of contract. 
This would make the tie to the RelayHub stronger (not constructor param, but actual address in the deployed code) - at the expense of a bit more complex deployment (require linking phase)

Reply:
I don't see a clear advantage in switching to libs here.

Reply:
The advantage is no configuration param, as the lib reference is hard-coded in the code.
There is not reason for us to switch now, as it adds deployment complexity, and very little security